### ISSUE TITLE
make kustomize doesn't install kustomize with go 1.18

### ISSUE THREAD

**Issue by jan--f** [rhobs/observability-operator#139]
go get  no longer builds a module:
```
In earlier versions of Go, 'go get' was used to build and install packages.
Now, 'go get' is dedicated to adjusting dependencies in go.mod. 'go install'
may be used to build and install commands instead.
```

**Comment by jan--f** [comment]
```
~ % go version
go version go1.18.1 linux/amd64
```

**Comment by jan--f** [comment]
This was fixed in #173.
